### PR TITLE
Fix sysconfig path

### PR DESCRIPTION
**Pull request by UnmeshDeodhar**
This PR reverts the change made by this PR: https://github.com/kata-containers/kata-containers/pull/7134/files#diff-2ba0cb0d4a795ecfd09b767b2c67bbaac58abaecc743ca4c1281966d940f699f
Adding SYSCONFIG_PATHS_SEV variable back.

Fixes: #7213

### CODE REVIEW

**Review by stevenhorsman — APPROVED** [review]
LGTM, sorry that I took the wrong option in the merge

File: src/runtime/Makefile [hunk 1]
```diff
@@ -347,7 +347,7 @@ ifneq (,$(QEMUCMD))
     CONFIG_PATHS += $(CONFIG_PATH_QEMU_SEV)
 
     SYSCONFIG_QEMU_SEV = $(abspath $(SYSCONFDIR)/$(CONFIG_FILE_QEMU_SEV))
-    SYSCONFIG_PATHS += $(SYSCONFIG_QEMU_SEV)
+	SYSCONFIG_PATHS_SEV += $(SYSCONFIG_QEMU_SEV)
```
Comment (fidencio) [inline src/runtime/Makefile:350]: Please, do not change the indentation here.

**Comment by wainersm** [comment]
The jenkins-ci-ubuntu-20.04-x86_64-cc_cri_containerd job is affected by https://github.com/kata-containers/tests/issues/5701

File: src/runtime/Makefile (hunk 1 above)
Comment (jepio) [reply src/runtime/Makefile]: I've pushed the fixup so that we can merge this

**Comment by jepio** [comment]
This PR is also included in https://github.com/kata-containers/kata-containers/pull/7205, so we'll merge that.

**Review by GabyCT — APPROVED** [review]

**Comment by stevenhorsman** [comment]
Closed as superceded by #7205

**Pull request closed without merging**